docker-compose.yml

```diff
@@ -16,6 +16,8 @@ services:
 
   arangodb:
     image: arangodb:latest
+    restart:
+      always
```

Comment:
brzmi niebezpiecznie

Reply:
to jest to co sprawi ze bedzie dzialac xd wczesniej na serwerze to sie usunelo i nie dzialalo przez to 👍